.eslintrc.base.js

```diff
@@ -63,5 +63,21 @@ module.exports = {
     // a browser context. For instance, we may import polyfills which change
     // global variables, or we may import stylesheets.
     'import/no-unassigned-import': 'off',
+
+    // import/no-named-as-default-member checks if default imports also have
+    // named exports matching properties used on the default import. Example:
+    // in confirm-seed-phrase-component.test.js we import sinon from 'sinon'
+    // and later access sinon.spy. spy is also exported from sinon directly and
+    // thus triggers the error. Turning this rule off to prevent churn when
```

Comment:
Can you go into more detail about what you mean about reducing churn when upgrading eslint? Do you mean that your editor slows to a crawl or something after eslint is upgraded? I guess I'm wondering if this is a temporary performance problem or a consistent one.

Reply:
Not performance - churn in code review. IE trying to minimize the amount of changes in one PR and focusing them on just prettier related changes for reviewing purposes

Reply:
Ohh gotcha, so this should be turned on, but we're disabling it for now since it's going to cause a lot more violations, is that what you mean?

Reply:
Exactly